src/databricks/labs/lsql/dashboards.py

```diff
@@ -604,10 +597,21 @@ def _get_query_widget_spec(fields: list[Field], *, frame: WidgetFrameSpec | None
 
 @dataclass
 class DashboardMetadata:
-    """The metadata defining a lakeview dashboard"""
+    """The metadata defining a lakeview dashboard
+
+    Attributes :
+        display_name : str
```

Comment:
my slight preference is to keep attribute docs close to attributes documented, otherwise we forget.